svgutil/size.go

```diff
@@ -0,0 +1,96 @@
+package svgutil
+
+import (
+	"encoding/xml"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	pixel = "px"
+
+	inch      = "in"
+	pxPerInch = float64(96)
+
+	centimeter      = "cm"
+	pxPerCentimeter = float64(37.79)
+
+	millimeter      = "mm"
+	pxPerMillimeter = float64(3.779)
+
+	point      = "pt"
+	pxPerPoint = float64(1.33)
+
+	pica      = "pc"
+	pxPerPica = float64(16)
+)
+
+type svg struct {
+	Width   string `xml:"width,attr"`
+	Height  string `xml:"height,attr"`
+	ViewBox string `xml:"viewBox,attr"`
+}
+
+func Size(str string) (width, height int, err error) {
+	parsed := svg{}
+	err = xml.Unmarshal([]byte(str), &parsed)
+	if err != nil {
+		return 0, 0, fmt.Errorf("%w: parse svg", err)
```

Comment:
```suggestion
		return 0, 0, fmt.Errorf("parse failed: %w", err)
```

nit. SVG 파싱에 실패했다라는 좀 더 포괄적인 개념을 앞으로 배치하고, 상세 오류 원인을 뒤쪽으로 배치하는게 어떨까요?